Process.c

```diff
@@ -85,8 +85,8 @@ static bool findCommInCmdline(const char* comm, const char* cmdline, int cmdline
 
       if ((tokenLen == commLen || (tokenLen > commLen && commLen == (TASK_COMM_LEN - 1))) &&
           strncmp(tokenBase, comm, commLen) == 0) {
-         *pCommStart = tokenBase - cmdline;
-         *pCommEnd = token - cmdline;
+         *pCommStart = (int)(tokenBase - cmdline);
+         *pCommEnd = (int)(token - cmdline);
```

Comment:
Given that these both refer to string offsets, we actually should switch to `size_t*` eventually (DiD).

Reply:
The type after the substraction is technically `ptrdiff_t`. Casting to `int` would lose upper bits. Also, it's a signed type.

Reply:
Technically yes, but since `tokenBase > cmdline` and `token > cmdline`, both are positive and a cast to `size_t` wouldn't cause trouble with the sign bit. I'd have to check re sentinel values, but that's another concern.